UI/html/blog.html

```diff
@@ -75,6 +75,7 @@ <h1>Subscribe to for news letter</h1>
         </ul>
     </section>
     <section id="all__blog">
+        <p id="deleteArticle"></p>
         <div class="all__blog">
             <div class="item" id="single__item">
                 <!--  <div class="article" id="blog__list"> -->
```

Comment:
Remove this line if you're not using it.

Reply:
Done